app/channels/change_task_channel.rb

```diff
@@ -10,8 +10,17 @@ def unfollow
 
   def change_task(data)
     data = data['data']
+    raise unless open_room data['room_id']
     title = "Task: #{data['title']}"
     ActionCable.server.broadcast("change_task:#{data['room_id']}", 
                                  { title: title, task_id: data['task_id'] })
   end
+
+  private
+
+  def open_room id
```

Comment:
Those 3 lines could be replaced by `SessionVote.update(id, status: :in_progress)`